### ISSUE TITLE
double response

### ISSUE THREAD

**Issue by brunodd** [burtenshaw/movie-chatter#7]
When starting the script and using the keywords 'about' and 'movie' in the same phrase, the chatterbot will first try to determine the movie and then immediately talk 'about' a proposed movie. I don't understand why both the movieAdapter and aboutAdapter are used. I would presume only 1 at a time can give a response before awaiting user input.

~~~
Type something to begin...
---
lets talk about movies
What's your favorite film? Maybe we can find something similar.
the godfather
Do you mean The Godfather directed by Francis Ford Coppola?
yes
Do you know Scarface?
yes
Something you might not know is ...
How about Scarface?
---
~~~

Note the part: 
~~~
Do you know Scarface?
yes
Something you might not know is ...
~~~
should not happen here.

**Comment by ttn114** [comment]
There are `raw_input()` and `print()` commands hardcoded into some of the adapters' `process()` methods, so even if an adapter's response isn't used (e.g. due to low confidence), the user still sees these prints.
Haven't figured out the best way to hold a dialog that spans multiple requests/replies using chatterbot yet. Perhaps a subclassed `ChatBot` that keeps a history of the conversation so the logic adapters can query this history?

**Comment by brunodd** [comment]
Yes of course. I should've caught that. Thanks for the clarification.
